src/client/java/teammates/client/scripts/ValidateAllTeamNameScript.java

```diff
@@ -0,0 +1,65 @@
+package teammates.client.scripts;
+
+import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import com.googlecode.objectify.Key;
+import com.googlecode.objectify.cmd.Query;
+
+import teammates.storage.entity.CourseStudent;
+
+
+
+
+/**
+ * Script to check if any team name is a email.
+ *
+ * <p>See issue #7285</p>
+ */
+public class ValidateAllTeamNameScript extends DataMigrationEntitiesBaseScript<CourseStudent> {
+
+    private final Pattern emailRegex =
```

Comment:
Should use `REGEX_EMAIL` in `FieldValidator`?

Reply:
I tried, but that would not pass lint check...

Reply:
Then just copy the regex.